projects/packages/search/postcss.config.js

```diff
@@ -0,0 +1,2 @@
+// Use Jetpack plugin's postcss config.
+module.exports = require( '../../plugins/jetpack/postcss.config' );
```

Comment:
I'm not sure if this is referenceable, coz it  crosses packages. May well just duplicate?

Reply:
In hindsight, this will likely fail once we distribute the package to the WPCOM vendor folder. I'll duplicate it for now, given that it's a small file.